package.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "node.js_course_practice",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

If you use `nodemon` for local development, better to add proper script command.

```suggestion
  "scripts": {
    "start:dev": "nodemon src/index.js"
  },
```
After it you can run it with `npm run start:dev` command.

---

@ihor-osadchyi-dev done